lib/src/components/NxAlert/NxAlert.tsx

```diff
@@ -21,22 +21,20 @@ export { Props, propTypes, NxAlertProps, nxAlertPropTypes } from './types';
  * @param className CSS class names to apply to the rendered component.
  * @param icon FontAwesome icon data to render
  */
-const NxAlert = forwardRef<HTMLDivElement, NxAlertProps>(
```

Comment:
`forwardRef` is no longer needed. `ref` can be specified as a prop like any other.